wolox-training/mr-rails
# Mailer

# Summary
* Added functionality to send email with last 10 articles

# Test plan
1. Create a configuration file on the root folder of the project named .config.yml with the Mandrill credentials
2. Go to the home page and click on the "Send last 10 articles by email" button
3. If you are not logged in you will be requested to do it
4. Check your email

.config.yml example

    mandrill:
      user_name: "user@example.com"
      password: "mandrill_password"
      domain: "example.com"

# Trello card
https://trello.com/c/ZftUUQgW/14-crear-email-con-los-ultimos-10-articulos-creados
